### PR TITLE
DeeperDeepDungeonDex v1.0.0.0

### PR DESCRIPTION
**Pull request by MidoriKami**
New plugin, DeeperDeepDungeonDex, this plugin allows you to browse the collection of Deep Dungeon mob information collected by the DeepDungeonCompendium ( https://www.ddcompendium.com/ ) in-game with a pleasing and easy to use user interface.

### CODE REVIEW

**Comment by MidoriKami** [comment]
I'm aware we are missing an Icon, we'll have one ready by the time PAC gets around to reviewing this plugin.

**Comment by MidoriKami** [comment]
A majority of the line count is from the contained datafile, we have a parser project that reads from the ddcompendium's github (with permission from the author), and parses the data in a more C# friendly json.

**Comment by MidoriKami** [comment]
Plugin is fully functional now with all included planned system/windows.

All that's left is minor bug fixes and data correction.

**Comment by MidoriKami** [comment]
Some minor bug fixes, boss floors weren't working quite right.

**Comment by goaaats** [comment]
bleatbot, rebuild

**Comment by philpax** [comment]
Apologies for the mixup, this plugin hasn't gotten through rules compliance yet. @ackwell's assuming responsibility for this and will aim to get it through ASAP; please direct any queries their way.

File: testing/live/DeeperDeepDungeonDex/manifest.toml
Comment (ackwell) [inline testing/live/DeeperDeepDungeonDex/manifest.toml:1]: nonblocking;
seems like a few of the folders have a mismatching `.System` namespace - assuming a restructure that wasn't updated in-code?

File: testing/live/DeeperDeepDungeonDex/manifest.toml
Comment (ackwell) [inline testing/live/DeeperDeepDungeonDex/manifest.toml:5]: **blocking;**
while i acknowledge the entertainment value of the name, pac is on the whole not comfortable with merging this as-stands, using a name that is clearly riffing on another plugin - regardless of circumstances.
i'm not going to ask yall to go through refactoring the entire plugin over this, but i _will_ ask you update user-facing strings, please.

**Review by ackwell — COMMENTED** [review]
code review pass lgtm. bunch of comments on the way through, only blocking thing is the name as i believe has been brought up prior. everything else is just commentary/pass-over/nitpicks.

File: testing/live/DeeperDeepDungeonDex/manifest.toml
Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: This was due to feature creep, initially I just wanted the smaller cropped images as thumbnails, then we found that it would be nice to show the full pictures on click.

Once I have more time I plan remove the thumbnails and calculate modified UV coordinates to display thumbnails.

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: I wanted to make sure we had the framework in place to allow for translations on our side in the case that the plugin became popular and translations become a popularly requested feature.

The upstream is a fairly old resource, and I did not want to impose on the maintainer more than I already was.

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: I believe it simpler than using territorytype id's, if they do add another Deep Dungeon to the game, I can easily add in the expansion ID before the territorytype id's are even available.

I doubt there's any difference in terms of stability.

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: The upstream data source only includes boss information via `floorset` entries, and they use different key values for the same fields that `enemy` uses. It is a hacky workaround. This is one thing that I'd work on improving before attempting to release to stable. Should be fine for testing though imo.

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: Yeah that's a copy paste error, rider auto-inserted an extra `\`

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: Nice catch, this has been fixed, thank you.

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:1]: As this is simply intended to check that the target has changed, I will adjust this to check for ObjectId instead.

File: testing/live/DeeperDeepDungeonDex/manifest.toml
Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:5]: I'll state upfront, I am willing to change the name to `[anything]DeepDungeonDex`.

WildWolf stated quite publicly that he intended to change the name of `DeepDungeonDex` (This can be seen in a public channel here: [XIVLauncherDiscord#DeepDungeonDex](https://discord.com/channels/581875019861328007/1019651246619099136/1195346556572930108)

By having some form of `DeepDungeonDex` in the name of this plugin then users that are interested only in Deep Dungeon content could easily find our plugin.

If this is insufficient then I would like to ask for a more real-time dialogue with PAC to discuss options.

Comment (ackwell) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:5]: heyo, i think the main issue we want to address is the direct inclusion of the existing plugin's name. we're by no means fortune tellers, but the potential for confusion or conflict over this is deemed undesireable.

there's no concern held with inclusion of "deep dungeon" in the title - as you say, users may search by content name, and including it in the title is valuable. the ask is _only_ to avoid the direct inclusion of "DeepDungeonDex" as-stands. some off-the-cuff names pac threw around at the time include "Deep Dungeon Helper" and "Of Dungeons Deep" (lmao i love it), but these are just some ideas.

while i doubt there's much to be gained from real time conversations bar reiteration of the above, by all means do reach out if you'd like to run ideas past - it's unlikely i'll specifically be around directly due to availability/timezones, but i'm fairly sure there'll be someone at most times.

it looks like someone's come through with a testing pass, so this looks good to go once naming is addressed

Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml:5]: This is acceptable. ODD will work.

**Review by philpax — APPROVED** [review]

File: testing/live/DeeperDeepDungeonDex/manifest.toml
Comment (MidoriKami) [reply testing/live/DeeperDeepDungeonDex/manifest.toml]: Addressed in v1.0.0.1

**Pull request merged**
